File: available/java/src/test/java/java/gradle/LibraryTest.java
package java.gradle;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class LibraryTest {
    @Test void testSomeLibraryMethod() {
        Library classUnderTest = new Library();
        assertTrue(classUnderTest.someLibraryMethod(), "someLibraryMethod should return 'true'");
    }
}

File: available/kotlin/bin/test/starters/LibraryTest.kt
package starters;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.Assertions.assertEquals;

class LibraryTest {

    @Test
    internal fun testSomeLibraryMethod() {
        val classUnderTest = Library();
        assertEquals(true, classUnderTest.someLibraryMethod());
    }
}
